esp/platform/espcontext.cpp

```diff
@@ -586,12 +586,28 @@ void CEspContext::updateTraceSummaryHeader()
     if (m_txSummary)
     {
         m_txSummary->set("activeReqs", m_active);
-        m_txSummary->set("user", VStringBuffer("%s%s%s", (queryUserId() ? queryUserId() : ""), (m_peer.length() ? "@" : ""), m_peer.str()).str());
+        VStringBuffer user("%s%s%s", (queryUserId() ? queryUserId() : ""), (m_peer.length() ? "@" : ""), m_peer.str());
+        if (!user.isEmpty())
+            m_txSummary->set("user", user.str());
 
-        VStringBuffer reqSummury("%s %s.%s", httpMethod.get(), m_servName.str(), servMethod.get());
+        VStringBuffer reqSummary("%s", httpMethod.isEmpty() ? "" : httpMethod.get());
```

Comment:
This is not HTTP specific. For example, a 'ping' message have an empty httpmethod.